# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1264189599 18000
#      Fri Jan 22 14:46:39 2010 -0500
# Node ID 8fa6ce1b21f236ff90403498e00acb97f9ed77cd
# Parent  97ae300aa73a7b59900867812500199855fec0bc
file-ops.cc: don't include statdefs.h

diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -1,8 +1,12 @@
+2010-01-22  John W. Eaton  <jwe@octave.org>
+
+	* file-ops.cc: Don't include statdefs.h.
+
 2010-01-22  Jaroslav Hajek  <highegg@gmail.com>
 
 	* dbleSVD.cc (SVD::init): Ensure LD* arguments >= 1.
 	* floatSVD.cc (FloatSVD::init): Likewise.
 	* CmplxSVD.cc (ComplexSVD::init): Likewise.
 	* fCmplxSVD.cc (FloatComplexSVD::init): Likewise.
 
 2010-01-21  John W. Eaton  <jwe@octave.org>
diff --git a/liboctave/file-ops.cc b/liboctave/file-ops.cc
--- a/liboctave/file-ops.cc
+++ b/liboctave/file-ops.cc
@@ -39,17 +39,16 @@ along with Octave; see the file COPYING.
 
 #include "dir-ops.h"
 #include "file-ops.h"
 #include "file-stat.h"
 #include "oct-env.h"
 #include "oct-passwd.h"
 #include "pathlen.h"
 #include "quit.h"
-#include "statdefs.h"
 #include "str-vec.h"
 #include "oct-locbuf.h"
 
 file_ops::static_members *file_ops::static_members::instance = 0;
 
 file_ops::static_members::static_members (void)
   :
 #if (defined (OCTAVE_HAVE_WINDOWS_FILESYSTEM) && ! defined (OCTAVE_HAVE_POSIX_FILESYSTEM))
